### ISSUE TITLE
Add support for environment variables

### ISSUE THREAD

**Issue by julioomoura** [EvidentSolutions/gradle-beanstalk-plugin#28]
Is is possible to set env variables to beanstalk through this plugin?

**Comment by komu** [comment]
No, it's not possible at the moment, but I suppose it would be quite easy to add support for that.

Personally I haven't used Beanstalk for a few years now, so I haven't done any new development to this plugin, I've just merged pull requests that people have sent. If you'll study Amazon's API how environment variables can be defined, and send a PR that does that, I can merge it as well.

Thanks for your question.
